### ISSUE TITLE
Add Chucker Flutter to inspects the HTTP(S) requests/responses

### ISSUE THREAD

**Issue by luongvo** [nimblehq/flutter-templates#221]
## Why

Inspired by [what we did in Android Templates](https://github.com/nimblehq/android-templates/pull/331), having a built-in Chucker tool that helps to inspect the HTTP(S) requests/responses is really helpful & efficient for the team, PM, and testers to determine and resolve the networking issues in the app build. We have the same tool in Flutter called [Chucker Flutter](https://pub.dev/packages/chucker_flutter), so it's time to add it to our template.
 
## Who Benefits?

Developers

**Comment by sleepylee** [comment]
not sure what we used in Varun but we definitely need this

**Comment by sleepylee** [comment]
yep I just found that it was alice as well.

**Comment by luongvo** [comment]
@sleepylee yeah, to be honest, we did not have any network inspector tool in the Varun project. Maybe in your `working project` @doannimble. After researching, Alice has more powerful features than Chucker Flutter with higher rating, so it would be more stable 😄

**Comment by sleepylee** [comment]
https://nimble-co.slack.com/archives/C01NKCFNE6R/p1636107577006200?thread_ts=1636107577.006200&cid=C01NKCFNE6R

**Comment by doannimble** [comment]
@luongvo We're using https://pub.dev/packages/inspector in easy Hotel Flutter project. I'm not sure if `Alice` supports GraphQL 🤔 
cc @sleepylee

**Comment by sleepylee** [comment]
@doannimble it needs to be detachable i.e on the Firebase/Testflight build without connecting to dev tool.
I think we still use dio under the hood of our graphql calls? I'm not sure but we can check it out. Remember that we don't always have graphql project, and this is supporting Dio as the first case

**Comment by luongvo** [comment]
> @luongvo We're using [pub.dev/packages/inspector](https://pub.dev/packages/inspector) in easy Hotel Flutter project. I'm not sure if `Alice` supports GraphQL 🤔 cc @sleepylee

it seems you sent a wrong tool @doannimble 😄

**Comment by doannimble** [comment]
@luongvo @sleepylee `Inspector` supports both `dio` and `graphQL`. In easy Hotel Flutter project, we are using both RESTful and GraphQL.
For GraphQL calls, we use `gql_link` to execute the request, not `dio`

**Comment by luongvo** [comment]
@doannimble do you mean this tool https://pub.dev/packages/requests_inspector?

**Comment by doannimble** [comment]
> @doannimble do you mean this tool https://pub.dev/packages/requests_inspector?

@luongvo  The similar one `https://pub.dev/packages/inspector`, not sure why there are two repo that have the same function/UI 😄

**Comment by luongvo** [comment]
Confirmed by @doannimble [here](https://nimble-co.slack.com/archives/C01NKCFNE6R/p1688960462997299?thread_ts=1688960011.890309&cid=C01NKCFNE6R) to use https://pub.dev/packages/requests_inspector for both HTTP with Dio and GraphQL with [gql_link](https://pub.dev/packages/gql_link).

**Comment by sleepylee** [comment]
in case we are still discussing with tool to use, it is open for discussion, thus I updated the title

**Comment by manh-t** [comment]
@luongvo I think we can create the voting so the members can choose the preferred tool. IMO, both request_inspector and Alice are easy to use, but I would choose Alice because it has more ratings on the Github page (more users use it so probably it's more stable 💡 )  😄
